packages/protect/__tests__/protect.test.ts

```diff
@@ -217,55 +118,80 @@ describe('bulk encryption', () => {
 // These tests pass locally, given you provide a valid JWT.
 // To manually test locally, uncomment the following lines and provide a valid JWT in the userJwt variable.
 // ------------------------
-// const userJwt = ''
+// const userJwt =
+//   ''
```

Comment:
Mind creating a card for incorporating the mock token server into this so we can test consistently?